python/docs/source/user_guide/sql/arrow_pandas.rst

```diff
@@ -387,7 +387,7 @@ working with timestamps in ``pandas_udf``\s to get the best performance, see
 Recommended Pandas and PyArrow Versions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For usage with pyspark.sql, the minimum supported versions of Pandas is 0.23.2 and PyArrow is 1.0.0.
+For usage with pyspark.sql, the minimum supported versions of Pandas is 1.0.5 and PyArrow is 1.0.0.
```

Comment:
Should we mention there are some issues with versions like 1.0.0, 1.0.1?

Reply:
How about?

For usage with pyspark.sql, the minimum supported versions of Pandas is 1.0.5 and PyArrow is 1.0.0. Lower versions (such as there are some known issue under with v1.0.0, v1.0.1, see more in [link](https://github.com/apache/spark/pull/34717)) or higher versions may be used, however, compatibility and data correctness can not be guaranteed and should be verified by the user.

Maybe need more suggestion from native speaker. T_T